cnudie/retrieve.py

```diff
@@ -661,7 +663,8 @@ def create_default_component_descriptor_lookup(
     cache_dir: str | None=None,
     oci_client: oc.Client=None,
     delivery_client=None,
-    default_absent_ok=False,
+    default_absent_ok: bool=False,
+    fallback_to_service_mapping: bool=True,
```

Comment:
I think we should document the semantics of fallback_to_service_mapping in docstr